File: src/main/java/com/example/speedyprintserver/ENTITY/Service/Service.java
package com.example.speedyprintserver.ENTITY.Service;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDate;

@Entity
@Table
@Data
public class Service {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idServices ;
    private String intitule ;
    private String description ;
    private String image ;
    private LocalDate date;
    //private Double prixConception ;
    //private Double prixImpression ;

    @ManyToOne(fetch = FetchType.LAZY)
    @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
    private CategorieService categorieService ;
}

File: speedyprintapp-webapp/src/app/core/Model/Parametre/Service.ts
import { CategorieService } from "./CategorieService";

export class Service {
  idServices ?:number ;
  intitule !:string ;
  description !:string ;
  image !:string;
  date !:string;
  forfait !:boolean;
  categorieService !:CategorieService;
}
